### ISSUE TITLE
Providing multiple "%s" parameters doesn't work as expected

### ISSUE THREAD

**Issue by macintacos** [evilpie/add-custom-search-engine#24]
I have a search engine defined in Chrome that is similar to the following:

```
https://example.com/?q=%s&somethingElse=%s
```

When I search for a value like "test" with this search engine in Chrome, I get the following (which, to me, feels like "expected behavior"):

```
https://example.com/?q=test&somethingElse=test
```

With this extension, only the first instance is replaced:

```
https://example.com/?q=test&somethingElse=%s
```

It'd be nice if this extension could replace all instances of %s that it finds in the URL that you provide for the search. Please let me know if you need any further clarification!

**Comment by evilpie** [comment]
Thanks for the report. Fixed by b99b46a3468f73bf86681795e051c7dfe9c47937.

**Comment by macintacos** [comment]
Awesome, thanks!
